### DISCUSSION TITLE
What Sources or Destinations should we add to Grouparoo?

### DISCUSSION THREAD

**Discussion by evantahler** [grouparoo/grouparoo#1071] [Chat]
We are always looking for what plugins we can make for Grouparoo to be more useful.  What should we connect to next?  If ideas get enough upvotes, they will be added to our [roadmap](https://github.com/orgs/grouparoo/projects/26)

**Comment by andyjih** [comment]
Mixpanel as a Destination: https://github.com/grouparoo/grouparoo/issues/1690

**Comment by andyjih** [comment]
Eloqua as a Destination: https://github.com/grouparoo/grouparoo/issues/1756

**Comment by andyjih** [comment]
Braze as a Destination: https://github.com/grouparoo/grouparoo/issues/650

**Comment by andyjih** [comment]
Google Sheets as a Destination: https://github.com/grouparoo/grouparoo/issues/1119

**Comment by andyjih** [comment]
Vero as a Destination: https://github.com/grouparoo/grouparoo/issues/1571

**Comment by andyjih** [comment]
Facebook Lookalike Audiences as a Destination: https://github.com/grouparoo/grouparoo/issues/1107

**Comment by andyjih** [comment]
GraphQL as a Source: https://github.com/grouparoo/grouparoo/issues/1413

**Comment by andyjih** [comment]
Klaviyo as a Destination: https://github.com/grouparoo/grouparoo/issues/1498

**Comment by andyjih** [comment]
Events (Amplitude, Mixpanel, Segment, etc.) as a Source: https://github.com/grouparoo/grouparoo/issues/661

**Comment by andyjih** [comment]
Segment as a Destination: https://github.com/grouparoo/grouparoo/issues/1629

**Comment by andyjih** [comment]
Amplitude as a Destination: https://github.com/grouparoo/grouparoo/issues/854

**Comment by andyjih** [comment]
Outreach as a Destination: https://github.com/grouparoo/grouparoo/issues/1146

**Comment by andyjih** [comment]
Drift as a Destination: https://github.com/grouparoo/grouparoo/issues/1147

**Comment by andyjih** [comment]
Salesforce Leads and Contacts as a Destination: https://github.com/grouparoo/grouparoo/issues/1392

**Comment by andyjih** [comment]
Salesforce as a Source: https://github.com/grouparoo/grouparoo/issues/1393

**Comment by andyjih** [comment]
Salesforce Marketing Cloud as a Destination: https://github.com/grouparoo/grouparoo/issues/1148

**Comment by andyjih** [comment]
Salesloft as a Destination: https://github.com/grouparoo/grouparoo/issues/1150

**Comment by andyjih** [comment]
Front as a Destination: https://github.com/grouparoo/grouparoo/issues/1151

**Comment by andyjih** [comment]
Salesforce Connect Integration: https://github.com/grouparoo/grouparoo/issues/1765

**Comment by andyjih** [comment]
Firebase (and Supabase) as a Source: https://github.com/grouparoo/grouparoo/issues/1242

**Comment by andyjih** [comment]
Close as a Destination: https://github.com/grouparoo/grouparoo/issues/1153

**Comment by andyjih** [comment]
Databricks as a Source: https://github.com/grouparoo/grouparoo/issues/1311

**Comment by andyjih** [comment]
Vertica as a Source: https://github.com/grouparoo/grouparoo/issues/1578

**Comment by andyjih** [comment]
Erxes as a Destination: https://github.com/grouparoo/grouparoo/issues/1375

**Comment by andyjih** [comment]
Mautic as a Destination: https://github.com/grouparoo/grouparoo/issues/1374

**Comment by andyjih** [comment]
Google Customer Match as a Destination: https://github.com/grouparoo/grouparoo/issues/1845

**Comment by jishnubiswas** [comment]
Amazon Athena as source!

**Reply by evantahler** [reply]
Card Added! https://github.com/grouparoo/grouparoo/issues/2043

**Reply by evantahler** [reply]
@jishnubiswas can you please share more about your use case and what kind of data you store in Athena?

**Comment by thusodangersimon** [comment]
Jira as a destination

**Reply by evantahler** [reply]
https://github.com/grouparoo/grouparoo/issues/2763

**Reply by andyjih** [reply]
Hi @thusodangersimon, and thanks for your comment! Could you share some more details of which Jira objects and endpoints specifically that you'd like to sync with? https://developer.atlassian.com/cloud/jira/platform/rest/v3/intro/? That'd be helpful for us to understand more about your use case.

**Comment by thusodangersimon** [comment]
freshdesk as a destination

**Reply by evantahler** [reply]
https://github.com/grouparoo/grouparoo/issues/2764

**Reply by andyjih** [reply]
Hi @thusodangersimon! Thanks for these comments. I've put a few thoughts on the issue linked above. Would you mind chiming in if there are specific data objects you're hoping to send to Freshdesk that aren't covered in the issue? Thanks!

**Reply by thusodangersimon** [reply]
Hi, I would like to be able to pull tickets and conversations from feshdesk. We are creating metrics for the completion of tickets. Supporting data about the user would also be great

**Comment by GAV1N** [comment]
NetSuite as a destination

**Reply by andyjih** [reply]
Thanks @GAV1N ! I've created an issue here https://github.com/grouparoo/grouparoo/issues/2778

I was wondering if you could describe a bit more about what data objects you're hoping to sync to in NetSuite?

**Reply by GAV1N** [reply]
Thanks, @andyjih! Apologies for being vague, but our NetSuite integration will only go live later this year so I'm not yet versed on the data objects we'll want to sync.
